src/idiomatic/foundations-api-design/predictable-api/naming-conventions/12-to.md

```diff
@@ -0,0 +1,29 @@
+---
+minutes: 2
+---
+
+# To
+
+Prefix to a function that takes a borrowed value and creates an owned value
+
+```rust
+String::to_owned // &str -> String (&str is not consumed)
+```
+
+<details>
+- Method for a function from a borrowed value to owned value.
+
+Not an escape hatch to make something borrowed into something owned, instead
+creates a new owned value.
```

Comment:
```suggestion
- We are not talking about some kind of borrow checker escape hatch, or unsafe code. We really mean creating a new owned value.
```

However this seems like a low-value remark to me, I'd be also OK with deleting it.